working-with-your-data/working-with-your-own-data.md

```diff
@@ -113,9 +113,51 @@ We don't recommend you put your password or any other credentials in the script
 ```
 wget --user=<USERNAME> --ask-password <URL>
 ```
-
 Using the `--ask-password` will prompt you to enter your password.
 
+### Upload data that is on a ssh server
+
+If you are retrieving your data from an [ssh server](https://searchsecurity.techtarget.com/definition/Secure-Shell), like one your institution or lab may host data on, we encourage you to use [the terminal command `scp`](https://linuxize.com/post/how-to-use-scp-command-to-securely-transfer-files/) to copy over your files you'd like to analyze to our server.
+(Make sure the data does not violate any of the [privacy issues described above](#things-to-know-before-uploading-your-data).)
+
+**Step 1)** Go to the Terminal tab in your RStudio session.
+
+![Terminal tab](screenshots/rstudio-session-terminal.png)
+
+**Step 2)** Copy over the [scp template script](./template-script/wget-TEMPLATE.sh).
+
+You'll find the `scp` template script in the `template-script/` folder.
+In the RStudio Server, you can click the check mark next to the file name, then go to `More` > and choose `Copy To`.
```

Comment:
As I said in the single comment, which was supposed to be part of this review, I don't think this script is helpful for `scp`, as I don't think it will work as intended. I think the single command instructions below are probably what we want.